### PR TITLE
Docs: Update Dynamic_Default_Fields.md

### PR DESCRIPTION
**Pull request by adrhumphreys**
Add some info about `default_records` and add a quick example of using `defaults`

### CODE REVIEW

File: docs/en/02_Developer_Guides/00_Model/How_Tos/Dynamic_Default_Fields.md [hunk 1]
```diff
@@ -1,7 +1,27 @@
-# Dynamic Default Values
+# Default Values and Records
 
-The [DataObject::$defaults](api:SilverStripe\ORM\DataObject::$defaults) array allows you to specify simple static values to be the default values when a
-record is created, but in many situations default values need to be dynamically calculated. In order to do this, the
+## Static Default Values
+The [DataObject::$defaults](api:SilverStripe\ORM\DataObject::$defaults) array allows you to specify simple static values to be the default values when a record is created
```
Comment (robbieaverill) [inline docs/en/02_Developer_Guides/00_Model/How_Tos/Dynamic_Default_Fields.md]:
```suggestion
The [DataObject::$defaults](api:SilverStripe\ORM\DataObject::$defaults) array allows you to specify simple static values to be the default values when a record is created.
```

File: docs/en/02_Developer_Guides/00_Model/How_Tos/Dynamic_Default_Fields.md [hunk 2]
```diff
@@ -1,7 +1,27 @@
...
+
+A simple example is if you have a dog and by default it's bark is "Woof"
```
Comment (robbieaverill) [inline docs/en/02_Developer_Guides/00_Model/How_Tos/Dynamic_Default_Fields.md]:
```suggestion
A simple example is if you have a dog and by default it's bark is "Woof":
```

File: docs/en/02_Developer_Guides/00_Model/How_Tos/Dynamic_Default_Fields.md [hunk 3]
```diff
@@ -38,3 +58,24 @@ public function populateDefaults()
     parent::populateDefaults();
 }
 ```
+
+## Static Default Records
+The [DataObject::$default_records](api:SilverStripe\ORM\DataObject::$default_records) array allows you to specify default records created on dev/build.
+
+A simple example of this is having a region model and wanting a list of regions created when the site is built
```
Comment (robbieaverill) [inline docs/en/02_Developer_Guides/00_Model/How_Tos/Dynamic_Default_Fields.md]:
```suggestion
A simple example of this is having a region model and wanting a list of regions created when the site is built:
```

File: docs/en/02_Developer_Guides/00_Model/How_Tos/Dynamic_Default_Fields.md [hunk 4]
```diff
@@ -38,3 +58,24 @@ public function populateDefaults()
...
+class Region extends DataObject 
+{
+    private static $db = [
+        'Title' => 'Varchar(45)',
+    ];
+    
+    private static $default_records = [
+        ['Title' => 'Auckland'],
+        ['Title' => 'Coromandel'],
+        ['Title' => 'Waikato']
```
Comment (robbieaverill) [inline docs/en/02_Developer_Guides/00_Model/How_Tos/Dynamic_Default_Fields.md]:
```suggestion
        ['Title' => 'Waikato'],
```

(code style only)

**Comment by adrhumphreys** [comment]
Dope, have just merged suggestions through Github, you should be able to just squash it all 👍

**Pull request merged**